dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -256,6 +256,13 @@ export const ImageComponent = ({
 		};
 		return images.slice().sort(descendingByWidth)[0].url;
 	};
+
+	// Its possible the tools wont send us any images urls
+	// if so, don't try to render
+	if (element.media.allImages.length === 0) {
```

Comment:
Is this worth moving to the top of the file as an exit early piece of code?

Reply:
Good point